app/src/main/res/values-fr-rFR/strings.xml

```diff
@@ -63,12 +63,12 @@
     <string name="start_radio">Démarrer la radio</string>
     <string name="play">Jouer</string>
     <string name="play_next">Jouer à la suite</string>
-    <string name="add_to_queue">Ajouter à la file d\’attente</string>
```

Comment:
I havent looked too closely but this is likely what it's complaining about. You need to include the `\` slash, because some special characters break the build system